docs/manage/_command_line.rst

```diff
@@ -155,12 +155,48 @@ If you need to restore a backup version prior to the latest one, you must specif
   This command is not intended for replication across different devices, but **only** for restoring on a single device from a local backup of the database.
 
 
+Change the Location of Kolibri Content Files
+--------------------------------------------
+
+Kolibri content channels may occupy a considerable amount of hard disk space over time. If you have concerns about running out of storage on your device, you can move just the Kolibri **content files** on another drive.
+
+.. tip::
+  If you have both SSD disk and HDD disk available on your device, it is recommended to install Kolibri on the SSD drive to allow faster access to the database, and move just the content file to the HDD drive.
+
+To move the Kolibri content folders to another location, follow these steps.
+
+1. Stop Kolibri.
+
+  .. code-block:: bash
+
+    kolibri stop
+
+
+2. Create a new folder that will contain all the content files and resources on the destination drive.
+
+  .. code-block:: bash
+
+    kolibri manage content movedirectory <destination>
+
+
+  For example, if you created a new folder ``KolibriContent`` on the drive ``F:``, run this command.
+
+  .. code-block:: bash
+
+    kolibri manage content movedirectory F:\KolibriContent
+
+3. Restart Kolibri.
+
+This command will move the 2 subfolders ``databases`` and ``storage``, from their default location inside the ``.kolibri/content`` folder in your device's ``Home`` directory, to a new location you specified in the command.
+
+
+
 Change the Location of ALL Kolibri Files
 ----------------------------------------
 
-If you want to change the directory where all of Kolibri’s runtime files go, the imported content channels, you need to change the environment variable called ``KOLIBRI_HOME`` to the path of your choice.
+If you want to change the directory where all of Kolibri's runtime files are located, together with the imported content channels, you need to change the environment variable called ``KOLIBRI_HOME`` to the path of your choice.
```

Comment:
This section might be misleading because it stresses "ALL Kolibri Files" and "together with the imported content channels". However, adjusting this environment variable behaves differently than the `movedirectory` command:

* Adjusting the environment variable will _not_ automatically migrate over data. You need to copy the _.kolibri_ folder manually to the new location
* If you _do_ copy the _.kolibri_ folder, the content directory will _not_ be affected if it had been previously set using the `movedirectory` command

Reply:
There's an unfortunate mix of "directory" and "folder" in the terminology. I know this was never consistent in the real world, just want to ask: Do we prefer any of them, or are we fine about mixing them?

Reply:
Personally I prefer the term `folder`, but I don't see a clear path to get out of the terminology "mix" for now... 😕 

Added Devon's suggestions in a note.